packages/neuron-wallet/tests-e2e/operations/sendTransaction.ts

```diff
@@ -0,0 +1,105 @@
+  const { client } = app.spectron
+
+  // Switch to send page
+  const receiveButton = await app.getElementByTagName('button', 'Send')
+  expect(receiveButton).not.toBeNull()
+  await client.elementIdClick(receiveButton!.ELEMENT)
+  await app.waitUntilLoaded()
+
+  // Get balance
+  let balance = 0
+  let retryCount = 5
+  while (balance === 0 && retryCount > 0) {
+    retryCount -= 1
+    const balanceElement = await app.element('//MAIN/DIV/DIV[4]/DIV[2]')
+    expect(balanceElement.value).not.toBeNull()
+    const balanceText = await client.elementIdText(balanceElement.value.ELEMENT)
+    balance = parseInt(balanceText.value.slice(0, balanceText.value.length - 4))
+    console.log(`balance = ${balance}`);
+    if (balance === 0) {
+      app.wait(1000)
+    }
+  }
+  // Check balance
+  let sendAmount = 0
+  for (let index = 0; index < sendToArr.length; index++) {
+    sendAmount += sendToArr[index].amount; 
+  }
+  if (sendAmount > balance) {
+    throw `The amount sent exceeds the balance. balance: ${balance} send_amount: ${sendAmount}`
+  }
+  
+  // Setup send elements
+  const sendItemGroupElementPath = '//MAIN/DIV/DIV[1]/DIV/DIV/DIV'
+  for (let index = 0; index < sendToArr.length; index++) {
+    const sendTo = sendToArr[index];
+    if (sendTo.amount < 61) {
+      throw `Amount should not be less than 61 CKB.  amount = ${sendTo.amount}`
+    }
+    const elementIndex = index + 1
+    // setup
+    const sendItemElementPath = `${sendItemGroupElementPath}/DIV[${elementIndex}]`
+    await app.setElementValue(`${sendItemElementPath}/DIV/DIV[1]//INPUT`, sendTo.address)
+    await app.setElementValue(`${sendItemElementPath}/DIV/DIV[2]//INPUT`, `${sendTo.amount}`)
+    console.log(`setup sent to ${sendTo.address} ${sendTo.amount}`);
+    if (elementIndex < sendToArr.length) {
+      // add
+      const addButton = await app.element(`${sendItemElementPath}/DIV/DIV[2]//BUTTON`)
+      expect(addButton.value).not.toBeNull()
+      await client.elementIdClick(addButton.value.ELEMENT)
+      app.waitUntilLoaded()
+    }
+  }
+
+  // Send
+  const sendButton = await app.element(`//MAIN/DIV/DIV[5]/BUTTON[2]`)
+  expect(sendButton.value).not.toBeNull()
+  await client.elementIdClick(sendButton.value.ELEMENT)
+  app.waitUntilLoaded()
+
+  // Input password
+  const passwordInputElement = await app.element('//BODY/DIV[4]/DIV/DIV/DIV/DIV[2]/DIV[2]/DIV/DIV[1]//INPUT')
+  expect(passwordInputElement.value).not.toBeNull()
+  await app.setElementValue('//BODY/DIV[4]/DIV/DIV/DIV/DIV[2]/DIV[2]/DIV/DIV[1]//INPUT', password)
+  // Confirm
+  const confirmElement = await app.element('//BODY/DIV[4]/DIV/DIV/DIV/DIV[2]/DIV[2]/DIV/DIV[2]/BUTTON[2]')
+  expect(confirmElement.value).not.toBeNull()
+  await client.elementIdClick(confirmElement.value.ELEMENT)
+  app.waitUntilLoaded()
+
+  // Transaction element
+  const transactionElementPath = `//MAIN/DIV/DIV[2]/DIV/DIV/DIV[2]/DIV/DIV/DIV/DIV/DIV/DIV[1]/DIV/DIV/DIV[2]/DIV/DIV/DIV[1]`
```

Comment:
This won't be easy to maintain in the future. There should be semantic class or id or other properties we can refer to UI elements. We can add that to the app if necessary.

Reply:
Ok, the element path is cumbersome to write and difficult to maintain after the change. I will add an id to the element for the search.